messages/auth.md

```diff
@@ -0,0 +1,37 @@
+# defaultOrgNotFound
+
+No %s org found
+
+# defaultOrgNotFoundActions
+
+- Run the "sfdx auth" commands with --setdefaultusername to connect to an org and set it as your default org.
+- Run "force:org:create" with --setdefaultusername to create a scratch org and set it as your default org.
+- Run "sfdx config:set defaultusername=<username>" to set your default username.
+
+# PortInUse
```

Comment:
tiny nit: for consistency I assume we should recommend camelCase?  i.e portInUse

Reply:
Ya, I can update these. Assuming everyone is ok with markdown, I can go update the rest of the messages too.